deployments/prod/environment.py

```diff
@@ -85,7 +372,7 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_PARTITION_PREFIX_LENGTH': '2',
 
         'AZUL_TDR_SOURCE_LOCATION': 'US',
-        'AZUL_TDR_SERVICE_URL': 'https://jade-terra.datarepo-prod.broadinstitute.org',
+        'AZUL_TDR_SERVICE_URL': 'https://data.terra.bio',
```

Comment:
Hannes updating this Service URL causes `make post_deploy` fail on the lungmap sources, I assume this is because the sources are not present in the new prod instance. 

Are there new snapshots for this do we need to ask the TDR team to create new snapshots in the updated instance?

Reply:
`azul.RequirementError: The service account (SA) 'azul-ucsc-0-prod@platform-hca-prod.iam.gserviceaccount.com' is not authorized to access snapshot 'hca_prod_1bdcecde16be420888f478cd2133d11d__20211013_20211013_lungmap' via the TDR API or that resource does not exist. Make sure that it exists, that the SA is registered with SAM and has been granted read access to the resource.`